ui/shared/components/panel/variants/VariantGene.jsx

```diff
@@ -475,11 +475,11 @@ const getDetailSections = (configs, gene, compact, labelProps, individualGeneDat
     ...acc,
     ...config.detail.map(detail => ({ ...config, ...detail })),
   ] : [...acc, config]),
-[]).map(({ detail, expandedDisplay, ...sectionConfig }) => (
-  (expandedDisplay && !compact) ? (
+[]).map(({ detail, expandedDisplay, expandedLabel, ...sectionConfig }) => (
+  (expandedDisplay && !compact && !noExpand) ? (
```

Comment:
it looks like `compact` is only used in this function in this line, so wouldn't it be cleaner to change what you pass as `compact` for this function instead of adding another prop that is also only used here?

Reply:
The `compact` is also used in a few lines below. It doesn't look okay if `compact` is set to `true`.
Is there any scenario in which we need to expand the `OMIM` details, and `showInlineDetails` is `true`? It seems not to me. So we could use `showInlineDetails` to disable expanding the `OMIM` details.